types/compiler.interfaces.ts

```diff
@@ -133,9 +133,14 @@ export type CompilerInfo = {
     removeEmptyGccDump?: boolean;
     irArg?: string[];
     minIrArgs?: string[];
-    optPipelineArg?: string[];
-    optPipelineModuleScopeArg?: string[];
-    optPipelineNoDiscardValueNamesArg?: string[];
+    optPipeline: {
```

Comment:
Or... `optPipeline:? {...the type here...}` Though I don't think I realised how many subfields it has...so maybe not :)